src/extract/__init__.py

```diff
@@ -5,3 +5,35 @@ def read_csv(filename):
             rows.append(line.strip().split(','))
     return rows
     
+
+def read_csv_nb(filename):
+    rows = []
+    empty_rows= 0
+    with open(filename, 'r') as f:
+        for line in f.readlines():
+            if line.strip() == ',,,,,':
+                empty_rows += 1
+            else:
+                rows.append(line.strip().split(','))
+    print('there are ' + str(empty_rows) + ' empty answers in the survey') 
+    return rows
+
+
+def read_csv_nb_CAP(filename):
+    rows = []
+    empty_rows= 0
+    with open(filename, 'r') as f:
+        for line in f.readlines():
+            if line.strip() == ',,,,,':
+                empty_rows += 1
+            else:
+                rows.append(line.strip().split(','))
+    print('there are ' + str(empty_rows) + ' empty answers in the survey') 
+
+    i = 0
+    while i < len(rows):
```

Comment:
Enhancement - This works, but could it be improved by looking at explicit referencing rather than numbered row? (i.e first_name, last_name)